client/components/place/place-list/place-list.component.js

```diff
@@ -107,6 +111,7 @@ angular.module('placeList', ['filterMapType', 'popularTracks', 'ngAnimate'])
         ctrl.hidePopularTracks = true;
 
         ctrl.checkPopularPlaces = function() {
+          angular.element('.popular-places-wrapper').css('display', 'block');
```

Comment:
[There is no ng-hide class](https://github.com/Social-projects-Rivne/GreenTourism/blob/8a10f3a42f3c9dd23dd6674dd2216fc0f26df647/client/components/place/place-list/place-list.template.html#L234) :) I mean, try `class="ng-hide ..."`, it's an Angular built in class.